L.Map.Sync.js

```diff
@@ -17,7 +17,7 @@
     // with the top right corner of the target map.
     // The values can be less than 0 or greater than 1. It will sync
     // points out of the map.
-    L.Util.offsetHelper = function (ratioRef, ratioTgt) {
```

Comment:
I think this is due to Leaflet's new rollup built system. I think moving it out of the utils again is the way to fix this.